src/Land/Model/RadiativeEnergyFlux.jl

```diff
@@ -0,0 +1,111 @@
+module RadiativeEnergyFlux
+
+using ...VariableTemplates
+using DocStringExtensions
+
+export AbstractNetSwFluxModel,
+    PrescribedSwFluxAndAlbedo,
+    PrescribedNetSwFlux,
+    compute_net_sw_flux,
+    compute_net_radiative_energy_flux
+
+"""
+   AbstractNetSwFluxModel{FT <: AbstractFloat}
+"""
+abstract type AbstractNetSwFluxModel{FT <: AbstractFloat} end
+
+"""
+    PrescribedSwFluxAndAlbedo{FT, FN1, FN2} <: AbstractNetSwFluxModel{FT}
+
+Structure which contains functions for shortwave albedo and
+shortwave fluxes. They are user-defined, constant across the domain
+and can be a function of time.
```

Comment:
Does it make sense to make albedo a function of time? Maybe the ERA5 data has this time series?

Reply:
I think so! Seems like albedo varies with time in ERA5. Here is what I found : 
https://www.ecmwf.int/sites/default/files/elibrary/2015/18490-radiation-quantities-ecmwf-model-and-mars.pdf

The albedos available from MARS are listed in Table 3. To interpret them we first discuss briefly how albedo is
calculated in the model (valid since model cycle CY32R2). The basic land albedo is calculated from a monthly
climatology derived from the MODIS satellite instrument, which has four components: albedos are specified
separately for wavelengths shorter than and longer than 0.7 µm, and separately for direct and diffuse radiation.
The quantities ALUVP, ALUVD, ALNIP and ALNID in Table 3 are the four components of the land-surface
albedo interpolated to the start time of the forecast from the monthly means bracketing this time in the MODIS
climatology. They do not include the effects of snow, and are only available at the start of the forecast.
At each timestep within the model, these four albedos are modified to add the contribution from lying snow,
which is a dynamic variable in the model. They are then used by the radiation scheme, and the fact that four
variables are used means that the total amount of radiation reflected from the surface will depend on cloud cover
(which affects the ratio between direct and diffuse surface radiation), trace gas concentrations (which affects the
ratio between surface downwelling radiation with wavelengths greater than and less than 0.7 µm) and solar zenith
angle (which affects both).
At each timestep an additional instantaneous broadband albedo is computed for diagnostic purposes only,
and stored in MARS as “FAL”, as indicated in Table 3. This variable includes the contribution from snow and seaice, but is computed as a weighted average of the diffuse albedos in the UV-visible and the near-infrared, weighted
according to the fixed top-of-atmosphere spectral distribution of incoming solar radiation rather than the variable

Reply:
ok! is it clear which albedo to use for driving e.g. the soil model?